### DISCUSSION TITLE
introduction,joining community for OUTREACHY'24 project: Develop a communications strategy for a growing MDAnalysis user and contributor base

### DISCUSSION THREAD

**Discussion by sangwan-smriti** [MDAnalysis/mdanalysis#4517] [Outreachy Discussions]
Hello everyone, hope you are having a great day!
Smriti this side, an OUTREACHY 2024 applicant, joining this community to contribute to the project titled-Develop a communications strategy for a growing MDAnalysis user and contributor base, mentored by @jennaswa 
Really excited to develop new communication strategies and help MDAnalysis reach a wider user base through more inclusive and easy-to-understand communication strategies. Hope my previous experiences in social media, writing newsletters and LinkedIn posts for past projects will be beneficial.

Also, our GitHub profile URL is required for the pre-proposal and I'm new to open source so how would it affect my pre-proposal?

I've gone through the code of conduct, installed MDAnalysis, and am still learning more about the work.
Any more links to relevant previous discussions and tips for pre-proposal are welcome:)

**Comment by tareema** [comment]
Hello Smriti! I'm having a good day and I hope you are as well. 

Regarding your question about your GitHub profile being new, I have no idea if/ how it'll affect your application. Hopefully, it doesn't.

I'd say you just need to familiarise yourself with MDAnalysis and start working on your pre-proposal as the deadline for submission is on **March 22nd** which is this Friday.

You can find a lot of previous discussions on the Discord server.

**Reply by sangwan-smriti** [reply]
thanks Tareema , i have started working on pre-proposal and joined discord server which was very helpful :)
